scripts/ci/examples/tfbuild.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -o pipefail
+set -o errexit
+
+pushd "$(dirname $(basename "${0}"))/examples/s3-full-access-policy" > /dev/null
```

Comment:
I'm adding some comments now for this.

Reply:
> Also, this script is named tfbuild.sh but is specific to one example. When we run CI for this, are we running a bunch of examples, or only a specific example in each job?

I can change the name. The other option is that when we have other Haskell based tests for `terraform-aws-foundation` we can build them in this script as well. However because we're running `tfbuild.sh` in the Gitlab CI yaml with a manual action for this example, it'd probably make more sense to rename it, keep it separate, and have different build scripts for different examples and tests.

Reply:
Right now the build step is using terraform to plan and apply the infrastructure needed for the test. So if something were to go wrong when we have more examples and tests, we'd need to address how to roll back all the infrastructure `tfbuild.sh` created.